imagine/likelihoods/ensemble_likelihood.py

```diff
@@ -60,30 +69,32 @@ def call(self, simulations_dict):
 
         if self._covariance_dict is None:
```

Comment:
I have still some reservations on why we allow for this case. The derivation of our likelihood clearly assumes a existing noise (gaussian) covariance. If we don't know the noise, we should be forced to make explicit assumptions on it. Right now we are implicitly assuming a unit noise standard deviation in whatever units the data has in this case. That means that the results of our inference are not invariant under unit transformations. (e.g. data of  say 5 **cm** would imply a std of 1 **cm**,  changing the  data to 0.05 **m** changes the error to 1 **m**!).

Reply:
I agree. It makes little sense to have a measurement without noise. I will remove this case and always require for a covariance dictionary to be provided.